### PR TITLE
Use CockpitLang cookie and Accept-Language for localization

### PR DESCRIPTION
**Pull request by stefwalter**
Instead of relying on AMD javascript module loaders to load the correct localized data, use more standard HTTP mechanisms.
    
Accept-Language is used by default. However since this is not driveable on a per-web-application basis, and javascript has no influence on it, we also allow a CockpitLang cookie to override the Accept-Language and cockpit-ws dereferences this header into Accept-Language if it exists.
    
cockpit-bridge now looks for files with the appropriate language in its extension name.
    
We continue to keep the sessionStorage setting of language for forwards compatibility, but it's no longer used.

### CODE REVIEW

**Comment by dperpeet** [comment]
needs rebase

File: pkg/shell/base_index.js [hunk 1]
```diff
@@ -685,15 +685,20 @@ define([
              */
             var manifest = local_manifests["shell"] || { };
             $(".display-language-menu").toggle(!!manifest.linguas);
+            var language = document.cookie.replace(/(?:(?:^|.*;\s*)CockpitLang\s*\=\s*([^;]*).*$)|^.*$/, "$1");
+            console.log("language", language);
```
Comment (petervo) [inline pkg/shell/base_index.js]: Still needed?

File: src/bridge/cockpitpackages.c [hunk 2]
```diff
@@ -850,7 +851,9 @@ handle_packages (CockpitWebServer *server,
       goto out;
     }
 
-  bytes = cockpit_web_response_negotiation (filename, package->paths, &chosen, &error);
+  languages = cockpit_web_server_parse_languages (headers, NULL);
+
+  bytes = cockpit_web_response_negotiation (filename, package->paths, languages[0], &chosen, &error);
```
Comment (petervo) [inline src/bridge/cockpitpackages.c]: Accept-Language headers can have multiple values, do we want to make an effort to fall back in the users preferred order?

**Comment by petervo** [comment]
Looks good, just one question if we want to support falling back. But that can be a follow up PR if you prefer.

File: src/bridge/cockpitpackages.c (hunk 2 above)
Comment (stefwalter) [inline src/bridge/cockpitpackages.c]: I don't think so. At least not for now. I've found no way to make this predictable. If we want to add this in the future then that shouldn't be a problem.

File: pkg/shell/base_index.js (hunk 1 above)
Comment (stefwalter) [inline pkg/shell/base_index.js]: Removed.

**Pull request closed without merging**